### PR TITLE
[zh-cn] Sync write-new-topic.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstream
```yaml
content/zh-cn/docs/contribute/style/write-new-topic.md
```

### CODE REVIEW

File: content/zh-cn/docs/contribute/style/write-new-topic.md [hunk 1]
```diff
@@ -52,20 +52,30 @@ Tutorial | A tutorial page shows how to accomplish a goal that ties together sev
...
+[Hugo archetypes](https://gohugo.io/content-management/archetypes/) to create
+new content pages. To create a new type of page, run `hugo new` with the path to the file
+you want to create. For example:
 -->
+### 创建一个新页面{#creating-a-new-page}
+
 为每个新页面选择其[内容类型](/zh-cn/docs/contribute/style/page-content-types/)。
-使用页面类型有助于确保给定类型的各主题之间保持一致。
+文档站提供了模版或 [Hugo 原型](https://gohugo.io/content-management/archetypes/)来创建新的内容页面。
+要创建新类型的页面，请使用要创建的文件的路径运行 `hugo new` 命令。例如：
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/write-new-topic.md:66]:
```suggestion
要创建新的页面类型，请在所要创建的文件的路径，运行 `hugo new` 命令。例如：
```

File: content/zh-cn/docs/contribute/style/write-new-topic.md [hunk 2]
```diff
@@ -52,20 +52,30 @@ Tutorial | A tutorial page shows how to accomplish a goal that ties together sev
...
+that you write. The docs site provides templates or
+[Hugo archetypes](https://gohugo.io/content-management/archetypes/) to create
+new content pages. To create a new type of page, run `hugo new` with the path to the file
+you want to create. For example:
 -->
+### 创建一个新页面{#creating-a-new-page}
+
 为每个新页面选择其[内容类型](/zh-cn/docs/contribute/style/page-content-types/)。
-使用页面类型有助于确保给定类型的各主题之间保持一致。
+文档站提供了模版或 [Hugo 原型](https://gohugo.io/content-management/archetypes/)来创建新的内容页面。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/write-new-topic.md:65]:
```suggestion
文档站提供了模板或 [Hugo Archetypes](https://gohugo.io/content-management/archetypes/) 来创建新的内容页面。
```

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**